src/qt/addresstablemodel.cpp

```diff
@@ -82,14 +82,14 @@ class AddressTablePriv
             LOCK(wallet->cs_wallet);
             for (const std::pair<CTxDestination, CAddressBookData>& item : wallet->mapAddressBook)
             {
-                const CBitcoinAddress& address = item.first;
```

Comment:
Interesting, I didn't know an implicit conversion could be done when extending the lifetime of a const ref. Scary!